pytorchvideo/data/labeled_video_paths.py

```diff
@@ -94,7 +94,7 @@ def from_directory(cls, dir_path: str) -> LabeledVideoPaths:
 
         # Find all classes based on directory names. These classes are then sorted and indexed
         # from 0 to the number of classes.
-        classes = sorted((f for f in pathlib.Path(dir_path).iterdir() if f.is_dir()))
+        classes = sorted((os.path.basename(f) for f in pathlib.Path(dir_path).iterdir() if f.is_dir()))
```

Comment:
This is a pathlib file so you should be able to just do this:

```suggestion
        classes = sorted((f.name for f in pathlib.Path(dir_path).iterdir() if f.is_dir()))
```

Reply:
Well!